ASIM/dev/Delete-SentinelFunction/Delete-LASavedFunction.ps1

```diff
@@ -0,0 +1,107 @@
+    <#
+        .SYNOPSIS
+        Deletes saved searches from Log Analytics workspace
+
+        .DESCRIPTION
+        Deletes functions from Log Analytics workspace.
+        Takes a list of strings for the function names.
+
+        .PARAMETER FunctionName
+        Specifies a comma delimited list of saved function name to be deleted. Accepts wildchars. 
+
+        .PARAMETER WorkspaceName
+        Specifies the workspace where functions are saved.
+
+        .PARAMETER ResourceGroup
+        Specifies the resource group of workspace.
+
+        
+        .PARAMETER Force
+        If set the user will not be prompted for approval. Be causious!.
+
+        .PARAMETER Category
+        Specifies the category of deleted functions.
+
+        .PARAMETER Category
+        Specifies which subscription (aka context) to use.
+
+
+        .EXAMPLE
+        PS> Delete-LASavedFunction DeleteM*, TestFunction* -Subscription "Contoso Production" -Workspace contosoc_ws -ResourceGroup soc_rg
+
+        .LINK
```

Comment:
What does Link mean? I would prefer it to be an aka.ms link.

Reply:
the link is displayed in the help for the function, and if you type Get-Help functionname **-online** it opens the link.
Sure, what aka should I use?

Reply:
aka.ms/ASimDelFunctionScript. Makes sense?

Reply:
sure. let's first decide on terminology